README.md

```diff
@@ -611,3 +615,65 @@ status:
   binding:
     name: kafka-eftg9
 ```
+
+## Role-Based Access Control (RBAC)
+
+Kubernetes clusters often utilize [Role-based access control (RBAC)][rbac] to authorize subjects to perform specific actions on resources. When operating in a cluster with RBAC enabled, the service binding reconciler needs permission to read resources that provisioned a service and write resources that services are projected into. This extension defines a means for third-party CRD authors and cluster operators to expose resources to the service binding reconciler. Cluster operators **MAY** impose additional access controls beyond RBAC.
+
+[rbac]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+
+### For Cluster Operators and CRD Authors
+
+Cluster operators and CRD authors **MAY** opt-in resources to service binding by defining a `ClusterRole` with a label matching `service.binding/controller=true`. For Provisioned Service-able resources the `get`, `list`, and `watch` verbs **MUST** be granted. For PodSpec-able resources the `get`, `list`, `watch`, `update`, and `patch` verbs **MUST** be granted.
+
+#### Example Resource
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: awesome-service-bindings
+  labels:
+    service.binding/controller: "true" # matches the aggregation rule selector
```

Comment:
For completeness sake, just capturing one of the discussion points from the interlock call - there was a question on whether of not it would make sense to use different labels for `ClusterRoles` concerned about the pod-specable applications vs the ones concerned about the bindable services. I believe, we got as far as saying that this wouldn't really make an immediate difference as long as we assume that the service binding reconciliation is being handled by a single controller. @arthurdm brought up a specific use case from their end where that isn't case. Personally, I'm good either way for now and don't see this question as a blocker for this PR. I'd hope things become more clear as soon as we have multiple implementations of this spec.